rclcpp/minimal_composition/src/publisher_node.cpp

```diff
@@ -12,18 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <chrono>
-
```

Comment:
Since the code is using `std::chrono_literals` it needs to keep this include.

Reply:
@dirk-thomas Thank you for reviewing. I've reverted it.

Because other cpp files don't also include it, I remove it.